doc/terminal-v2-roadmap.md

```diff
@@ -0,0 +1,76 @@
+| 2020-11-30 | 1.6 in Windows Terminal Preview<br>1.5 in Windows Terminal | |
+| 2020-12-31 | 1.7 in Windows Terminal Preview<br>1.6 in Windows Terminal | |
+| 2021-01-31 | 1.8 in Windows Terminal Preview<br>1.7 in Windows Terminal | |
+| 2021-02-28 | 1.9 in Windows Terminal Preview<br>1.8 in Windows Terminal | |
+| 2021-03-31 | 1.10 in Windows Terminal Preview<br>1.9 in Windows Terminal | |
+| 2021-04-30 | 2.0 RC in Windows Terminal Preview<br>2.0 RC in Windows Terminal | |
+| 2021-05-31 | [2.0](https://github.com/microsoft/terminal/milestone/22) in Windows Terminal Preview<br>[2.0](https://github.com/microsoft/terminal/milestone/22) in Windows Terminal | |
+
+## Issue Triage & Prioritization
+
+Incoming issues/asks/etc. are triaged several times a week, labeled appropriately, and assigned to a milestone in priority order:
+
+* P0 (serious crashes, data loss, etc.) issues are scheduled to be dealt with ASAP
+* P1/2 issues/features/asks  assigned to the current or future milestone, or to the [Terminal 2.0 milestone](https://github.com/microsoft/terminal/milestone/22) for future assignment, if required to deliver a 2.0 feature
+* Issues/features/asks not on our list of 2.0 features are assigned to the [Terminal Backlog](https://github.com/microsoft/terminal/milestone/7) for subsequent triage, prioritization & scheduling.
+
+## 2.0 Scenarios
+
+The following are a list of the key scenarios we're aiming to deliver for Terminal 2.0.
+
+> 👉 Note: There are many other features that don't fit within 2.0, but will be re-assessed and prioritized for 3.0, the plan for which will be published in 2021.
+
+| Priority\* | Scenario | Description/Notes |
+| ---------- | -------- | ----------------- |
+| 0 | Settings UI | A user interface that connects to settings.json. This provides a way for people to edit their settings without having to edit a JSON file.<br><br>Issue: [#1564] |
+| 0 | Command palette | A popup menu to list possible actions and commands.<br><br>Issues: [#5400], [#2046]<br>Spec: [#2193] |
+| 1 | Tab tear-off | The ability to tear a tab out of the current window and spawn a new window or attach it to a separate window.<br><br>Issue: [#1256]<br>Spec: [#2080] |
+| 1 | Clickable links | Hyperlinking any links that appear in the text buffer. When clicking on the link, the link will open in your default browser.<br><br>Issue: [#574] |
+| 1 | Default terminal | If a command-line application is spawned, it should open in Windows Terminal (if installed)\*.<br>\* Or your preferred terminal<br><br>Issue: [#492]<br>Spec: [#2080] |
+| 1 | Overall theme support | Tab coloring, title bar coloring, pane border coloring, pane border width, definition of what makes a theme<br><br>Issue: [#3327]<br>Spec: [#5772] |
+| 1 | Open tab as admin/other user | Open tab in existing Windows Terminal instance as admin (if Terminal was run unelevated) or as another user.<br><br>Issue: [#5000] |
+| 1 | Traditional opacity | Don't add the acrylic blur.<br><br>Issue: [#603] |

```

Comment:
Rewrite this description. Telling me what a feature doesn't do doesn't actually tell me what it does hahaha.